examples/bedrock-agentcore-multi-agent/config.yaml

```
@@ -0,0 +1,250 @@
+  # Product Specialist Agent - handles product recommendations
+  - id: product-agent
+    bedrock:agentcore:PRODUCT_AGENT_ID
+    config:
+      agentId: PRODUCT_AGENT_ID
+      agentAliasId: PRODUCT_ALIAS_ID
+      region: us-east-1
+      enableTrace: true
+      memoryConfig:
+        type: short-term
+        enabled: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)